Library/Homebrew/language/node.rb

```diff
@@ -1,10 +1,40 @@
+require "json"
+
 module Language
   module Node
     def self.npm_cache_config
       "cache=#{HOMEBREW_CACHE}/npm_cache"
     end
 
-    def self.pack_for_installation
+    def self.pack_for_installation(prepare_required: false)
+      # Doing this unconditional could break for already prepared npm registry
+      # tarballs, because our local npm install would unecessary call prepare /
+      # prepublish a second time on them which might fail (e.g. calling mkdir
+      # on an already existing folder).
+      # We also can't just always do the local npm install und bundledDeps into
+      # package trick and just pass --ignore-scripts to the local npm install
```

Comment:
```
`--ignore-scripts`
```

and generally do this for all referenced variable/parameter names and other flags.